src/components/Links/SortLink.tsx

```diff
@@ -0,0 +1,32 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  sortFilter: string,
+};
+
+export const SortLink: React.FC<Props> = ({ sortFilter }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get(FilterType.SORT) || null;
+  const order = searchParams.get(FilterType.ORDER);
```

Comment:
It is a good practice to provide a default value for the 'order' variable in case it is not found in the searchParams. You can add '|| null' or default value you want to be set, at the end of the line, like you did for the 'sort' variable.

Reply:
added